openeo-geotrellis/src/main/scala/org/openeo/geotrellis/netcdf/NetCDFRDDWriter.scala

```diff
@@ -157,7 +197,7 @@ object NetCDFRDDWriter {
       }else{
         path
       }
-
+    val histograms = collection.mutable.Map[String,(StreamingHistogram,Int)]()
```

Comment:
```suggestion
    val bandHistograms = collection.mutable.Map[String, (StreamingHistogram, Int)]()
```

Unclear that the keys are band names.